src/getExtensionApi.ts

```diff
@@ -26,7 +26,7 @@ export async function getAzureContainerAppsApi(context: IActionContext, installM
     const acaExtension: apiUtils.AzureExtensionApiProvider | undefined = await apiUtils.getExtensionExports(acaExtensionId);
 
     if (acaExtension) {
-        return acaExtension.getApi<acaApi.AzureContainerAppsExtensionApi>('~0.0.1');
+        return acaExtension.getApi<acaApi.AzureContainerAppsExtensionApi>('^1.0.0');
```

Comment:
Well... technically, it doesn't need 1.0.1; it only needed 0.0.1 since they're backwards compat. But because I don't think ^0.0.1 would actually work, I just set it at ^1.0.0 to sort of emulate it being at the old version? idk if that makes sense lol